gilbreth_perception/README.md

```diff
@@ -1,5 +1,14 @@
-1.Save gilbreth_perception folder under your-catkin-workspace/src.
+For using Correspondence Grouping method in object recognition:
+1.Run command "roslaunch gilbreth_perception gilbreth_perception.launch cnn:=false".
 
-2.Under your-catkin-workspace run command "catkin_make".
-
-3.Run command "roslaunch gilbreth_perception gilbreth_perception.launch".
+For using convolutional neural network method in object recognition:
+1. Make sure your computer has Nvidia GPU;
```

Comment:
I read that thread and it isn't very clear whether or not it can be used on a machined equipped with an nvidia GPU.  Either way I'll find one and test,  which Cuda library did you install, can you provide a link to the installation instruction for that?

Reply:
I use CUDA version 8.0 instead of 9.0
https://developer.nvidia.com/cuda-80-ga2-download-archive

Reply:
Ok, I'll install this one then.

Reply:
I installed the cuda library you pointed to on a PC with an nvidia Quadro K620 , I also set the **THEANO_FLAGS** variable, however when I ran the launch file 
```
roslaunch gilbreth_perception gilbreth_perception.launch
```
I still kept getting this error:
```
TypeError: CudaNdarrayType only supports dtype float32 for now. Tried using dtype float64 for variable None
```

Reply:
Please refer to here: https://github.com/dimatura/voxnet/issues/5, 
The issue can be resolved using a .theanorc file in the home directory.
.theanorc
[global]
floatX = float32
device = gpu